collectionstyles.css

```diff
@@ -0,0 +1,32 @@
+main {
+    background-color: lightblue;
+    display: flex;
+    margin-top: 300px;
+    flex-wrap: wrap;
+
+}
+
+div {
+    width: 33%;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+}
+
+
+img {
+    width: 150px;
+}
+
+@media screen and (max-width: 850px) {
```

Comment:
Could display better at lower resolutions, I tested on iPhoneSE and its as bit small
<img width="825" alt="image" src="https://user-images.githubusercontent.com/65457376/217850210-1c0e9d1a-42a8-40bc-8dfe-b6ee45867c21.png">